### PR TITLE
Add gax/ruby template

### PR DESCRIPTION
**Pull request by jmuk**
This modifies packager.js to support README.md.mustache which
does not exist in templates/ruby.

### CODE REVIEW

**Comment by tbetbetbe** [comment]
LGTM 👍

File: templates/gax/ruby/PUBLISHING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+= PUBLISHING
+
+Note: This folder has been generated by the api-packager tool.
```
Comment (tbetbetbe) [inline templates/gax/ruby/PUBLISHING.md]: s/api-packager/gen-api-package/

Comment (jmuk) [inline templates/gax/ruby/PUBLISHING.md]: done

File: templates/gax/ruby/README.md.mustache [hunk 2]
```diff
@@ -0,0 +1,36 @@
+GAX library for the Google {{api.simplename}} API
+=================================================
+
+{{api.name}}-{{api.version}} uses [google-gax][Google API extensions] to provide an
+easy-to-use client library for the [`Google {{api.simplename}} API`][] ({{api.version}}) defined in the [googleapis][] git repository
```
Comment (tbetbetbe) [inline templates/gax/ruby/README.md.mustache]: FYI: I recommend s/api.simplename/api.titlename/ for use in text now that #26 is checked in.

The main difference is that titlename is capitalized, the resulting text reads a bit better.

File: templates/gax/ruby/README.md.mustache [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+[googleapis]: https://github.com/googleapis/googleapis/tree/master/google/{{api.path}}/{{api.version}}
+[google-gax]: https://github.com/googleapis/gax-ruby
+[`Google {{api.simplename}} API`]: https://cloud.google.com/{{api.simplename}}/
+
+Getting started
+---------------
+
+{{api.name}}-{{api.version}} will allow you to connect to the [`Google {{api.simplename}} API`][] and access all its methods. In order to achieve so you need to set up authentication as well as install the library locally.
```
Comment (tbetbetbe) [inline templates/gax/ruby/README.md.mustache]: s/api.simplename/api.titlename/ for the text

File: templates/gax/ruby/README.md.mustache [hunk 4]
```diff
@@ -0,0 +1,36 @@
+GAX library for the Google {{api.simplename}} API
+=================================================
+
+{{api.name}}-{{api.version}} uses [google-gax][Google API extensions] to provide an
+easy-to-use client library for the [`Google {{api.simplename}} API`][] ({{api.version}}) defined in the [googleapis][] git repository
+
+
+[googleapis]: https://github.com/googleapis/googleapis/tree/master/google/{{api.path}}/{{api.version}}
+[google-gax]: https://github.com/googleapis/gax-ruby
+[`Google {{api.simplename}} API`]: https://cloud.google.com/{{api.simplename}}/
```
Comment (tbetbetbe) [inline templates/gax/ruby/README.md.mustache]: s/api.simplename/api.titlename/ for the text but not for the uri

File: templates/gax/ruby/README.md.mustache (hunk 2 above)
Comment (jmuk) [inline templates/gax/ruby/README.md.mustache]: Yup, I did that in the next patch actually 😉

**Comment by tbetbetbe** [comment]
LGTM

**Pull request merged**